doc/selfhosting.md

```diff
@@ -123,23 +123,31 @@ API. Generate one randomly with `pwgen -s 30 1`.
 ## Build AMI
 
 You'll want to run `make env` to load in the new variables from
-`.env`. Now run `make packer`. This will take up to 10 minutes to
-build a timestamped AMI with a name like `riju-20210711223158`.
+`.env`. Now run `make packer-ci`. This will take up to 5 minutes to
+build a timestamped AMI with a name like `riju-ci-20210711223158`.
+Then run `make packer-web`. This will take up to 10 minutes to build
+another AMI with a name like `riju-web-20210711223158.`
 
 ## Create local configuration (part 2 of 3)
 
 Add to `.env` the following contents:
 
 ```
 # Terraform
-AMI_NAME=riju-20210711223158
+AMI_NAME=riju-web-20210711223158
+CI_AMI_NAME=riju-ci-20210711223158
 AWS_REGION=us-west-1
-S3_BUCKET=yourname-riju
+S3_BUCKET=riju-yourname-tf
+SSH_KEY_NAME=
```

Comment:
`SSH_KEY_NAME` is no longer used because I got rid of the dev server (cost reasons). The same function will be provided in a more cost-effective manner with the improvements I have planned for CI; in the meantime, if you don't have symmetric upload on your Internet plan, the new self-hosting directions suggest manually provisioning an EC2 instance to do the build and upload on.

I've gotten rid of references to `SSH_KEY_NAME` generally, since I've migrated everything to use EC2 Instance Connect instead. One less thing to manually set up.